### DISCUSSION TITLE
Element Plus Table with virtual async load ?

### DISCUSSION THREAD

**Discussion by orenmizr** [element-plus/element-plus#4249] [Q&A] [answered]
hey friends, i truly believe this is the best UI lib out there right now. i know there is a plan for virtual table in the road map. 
does anyone know it's release day ? 

@jeremywu

**Comment by laizuan** [comment]
I hope to use the virtual table soon, even the beat version.  But I don't know when it will be released.

**Comment by ExtReMLapin** [comment, marked as answer]
https://github.com/element-plus/element-plus/tree/feat/virtualized-table

**Reply by orenmizr** [reply]
wow. the team behind this library is one of the best ones i've seen 🙏. 

do you know the status of it ? is it a dev branch ? is it on a npm package as well or the only way to try it is to clone the repo.

**Reply by ExtReMLapin** [reply]
clone the repo and give a try, there is no documentation yet i think
